VitalDevices/src/main/java/io/tryvital/vitaldevices/devices/GATTMeter.kt

```diff
@@ -136,9 +165,17 @@ abstract class GATTMeter<Sample>(
     }
 
     @SuppressLint("MissingPermission")
-    private fun bond() {
-        ensureBond().fail { _, status -> logError("bond", status) }
-            .done { vitalLogger.logI("Bonded with ${scannedDevice.name}") }.enqueue()
+    private fun bond(onDone: () -> Unit, onFail: (Int) -> Unit) {
+        ensureBond()
+            .fail { _, status ->
+                logError("bond", status)
```

Comment:
Sure about logging this without the vitalLogger?

Reply:
happy to ditch it

Reply:
It's just because in a different line, we are using `vitalLogger` directly. Maybe we should be consistent in its usage?